### PR TITLE
change function invocation for new arrow function

### PR DESCRIPTION
**Pull request by alikadir**
*Issue #, if available:*
function invoke not working 🙁

*Description of changes:*
I have been changed function invocation for following arrow function template 
[arrow function template file](https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-category-function/provider-utils/awscloudformation/function-template-dir/index.js.ejs)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-frontend-javascript/lib/aws_exports.js.ejs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // WARNING: DO NOT EDIT. This file is automatically generated by AWS Amplify. It will be overwritten.
 
-const awsmobile = <%- JSON.stringify(props.configOutput, null, 4) %>;
+const awsconfig = <%- JSON.stringify(props.configOutput, null, 4) %>;
```
Comment (kaustavghosh06) [inline packages/amplify-frontend-javascript/lib/aws_exports.js.ejs]: Why is this change needed?

Comment (alikadir) [reply packages/amplify-frontend-javascript/lib/aws_exports.js.ejs]: because the code is in the my web project. I think, the variable name should be more inclusive.

Comment (kaustavghosh06) [reply packages/amplify-frontend-javascript/lib/aws_exports.js.ejs]: Unfortunately a lot of our blogs and documentation refer to this variable name, so we can't make this change.

Comment (alikadir) [reply packages/amplify-frontend-javascript/lib/aws_exports.js.ejs]: I have reverted aws_exports.js.ejs.
Maybe later we make extensive change for variable names

**Comment by edwardfoyle** [comment]
Hi @alikadir, with the latest CLI release (4, we have significantly refactored local function invocation. Can you check if the changes solve this problem for you?

**Comment by alikadir** [comment]
hi @edwardfoyle 
I've tried local function invocation with CLI 4.18 but unfortunately my problem is not resolved. 😔

Local function Invocation has sent event parameter (json file) as string 
However it should send event parameter (json file) as object because aws console works this way

```javascript
exports.handler = async event => {
    return typeof event
}
// local invocation via amplify cli = "string"
// upload lambda function and test with event template on aws console = "object" 
```

**Comment by alikadir** [comment]
by the way, how to do debug the function during local invocation?
```shell
amplify mock function DisasterApiLogin --inspect=56884 --debug-brk //not working
```

**Comment by edwardfoyle** [comment]
@alikadir Ah yeah looks like you're right. It was a super small fix so I went ahead and made a PR for it here: https://github.com/aws-amplify/amplify-cli/pull/3895

**Comment by edwardfoyle** [comment]
closing this as #3895 has been merged
